README.md

```diff
@@ -61,13 +61,13 @@ One can open or check existing issues on [GitHub issues](https://github.com/bale
 
 ![Improve-this-doc-button](https://user-images.githubusercontent.com/22801822/113422793-ee9f9c00-93ea-11eb-90de-539e3438babb.png)
 
-## Deploying
+If the feature/change needs to be reviewed, a member of the balena team can/will deploy the PR to staging to check the changes. If everything looks good, then the PR would be merged into the master branch promptly in line for the changes to be deployed to production.
 
-After a PR has been created, a member of the balena team will review and merge the PR. The pull request will undergo CI/CD checks to verify if everything is correct. For example: if a Change-type has been added. More information present in the [CONTRIBUTING document](CONTRIBUTING.md)
+## Deploying
 
-Shortly after merging the PR to the `master` branch, it'll automatically be deployed to our [production environment](https://www.balena.io/docs/) (This should only take a few minutes).
+Shortly after merging a PR to the `master` branch, the Heroku app which hosts the [production instance](https://www.balena.io/docs/) will pick up the changes and deploy them. This should only take a few minutes.
 
-If the feature/change needs to be reviewed, a member of the balena team can/will deploy the PR to staging to check the changes. If everything looks good, then the PR would be merged into the `master` branch promptly in line for the changes to be deployed to production.
+If changes are needed to modify this deployment strategy message operations for access to the `resin-docs` app in Balena's Heroku team.
```

Comment:
At this point you seemed to switch from a public-facing message to an internal only context.

Reply:
Agreed. The part I removed I felt isn't necessary about how we will review a PR and the part I wrote is a new addition that just happens to be on the same line (they are not intended to be related). My added text is to convey how deployments work which is an internal process.

Do you have any ideas how we should document how docs are deployed ?

Reply:
I think deleting this line altogether will be fine to remove the redundant stuff about reviews, and skip the internal bit about contacting ops.